support/mac/install_dev_9_mac.sh

```diff
@@ -3,7 +3,7 @@ set -eux
 
 # Install Rust
 curl -sSf https://sh.rustup.rs | sh -s -- -y --default-toolchain stable
-. $HOME/.cargo/env
```

Comment:
Why are the double quotes only around `$HOME` here, but they're around the entire line in the file above, `install_dev_9_linux.sh`?